apps/scan/backend/tsconfig.build.json

```diff
@@ -10,7 +10,6 @@
     "declarationMap": true
   },
   "references": [
-    { "path": "../../../libs/api/tsconfig.build.json" },
```

Comment:
places where we reference build dependencies that are no longer _actual_ dependencies break during typescript build, because it can't find the relevant node module